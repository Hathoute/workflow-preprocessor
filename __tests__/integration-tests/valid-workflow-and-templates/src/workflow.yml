name: Workflow
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main

imports:
  - templates/template.yml
  - templates/template2.yml

jobs:
  build:
    extends: template/build
  test:
    extends: template2/test