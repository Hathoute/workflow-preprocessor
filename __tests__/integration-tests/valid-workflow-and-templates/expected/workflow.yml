name: Workflow
'on':
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
jobs:
  build:
    runs-on: ubuntu-18.04
    name: A job to build
    steps:
      - name: Build
        run: |
          echo "Building the app"
          ./build-script.sh
          echo "Finished building"
  test:
    runs-on: ubuntu-18.04
    name: A job to test
    steps:
      - name: Test
        run: |
          echo "Testing the app"
          ./test-script.sh
          echo "Finished testing"
